src/internal/m365/onedrive/item_handler.go

```diff
@@ -147,11 +147,11 @@ func (h itemRestoreHandler) AugmentItemInfo(
 	return augmentItemInfo(dii, item, size, parentPath)
 }
 
-func (h itemRestoreHandler) NewItemContentUpload(
+func (h itemRestoreHandler) DeleteItem(
 	ctx context.Context,
 	driveID, itemID string,
-) (models.UploadSessionable, error) {
-	return h.ac.NewItemContentUpload(ctx, driveID, itemID)
+) error {
+	return h.ac.DeleteItem(ctx, driveID, itemID)
```

Comment:
I kind of forgot about this until now, but do we need to start worrying about the whole chunked response on delete calls issue that was causing CI to fail for a while? Or was that deleting folders only?

Reply:
You mean the empty response body after a delete?  Shouldn't need to worry, that protection is supposed to be built into the api layer funcs (and it is here).